TACC/Core-Portal
# task/FP-331: add user guide to local development setup

## Overview: ##
Adds the Frontera User Guide to the local `docker-compose.dev.all.debug` for local development and testing.

## PR Status: ##


## Related Jira tickets: ##

* [FP-331](https://jira.tacc.utexas.edu/browse/FP-331)

## Testing Steps: ##
1. run `make start`
2. navigate to `https://cep.dev/user-guide` and confirm user guide is rendered works correctly
